README.adoc

```diff
@@ -131,6 +131,24 @@ This can be achived with the {uri-kube-apiserver}[Kubernetes API server CLI], e.
 $ kube-apiserver --cors-allowed-origins .*
 ```
 
+== Authentication
+
+We try to support the various authentication strategies supported by https://kubernetes.io/docs/reference/kubectl/overview/[kubectl] in order to provide seamless integration with your local setup. Here are the different authentication strategies we support depending on how you're using Kubebox:
+
+[cols="<,^,^,^,^,^", options="header"]
+|===
+||Executable |Server |Kubectl | Docker| Online
```

Comment:
I'm not sure whether this is really applicable to the _server_ and _kubectl_ modes, as they use service accounts to authenticate and do not rely on kube config?

Reply:
You're right! Updated